core/domain/summary_services.py

```diff
@@ -317,7 +322,8 @@ def get_displayable_exp_summary_dicts_matching_ids(exploration_ids, user=None):
 
     filtered_exploration_summaries = []
     for (exploration_summary, exploration_rights) in (
-            zip(exploration_summaries, exploration_rights_objects)):
+            list(python_utils.ZIP(
```

Comment:
No need for list here, just use ZIP directly.

Reply:
Done